library/net.cpp

```diff
@@ -373,38 +398,26 @@ static bool dnet_addr_is_local(dnet_node *n, const dnet_addr *addr)
  *
  * All sockets are sorted by their address, so we are able quickly to lookup if there are already such sockets.
  */
-static dnet_addr_socket_list *dnet_socket_create_addresses(dnet_node *node, const dnet_addr *addrs, size_t addrs_count,
```

Comment:
Do you mean about method of result returning or about used data structure (list vs set)?